runtime/shiden/src/lib.rs

```diff
@@ -1284,9 +1284,22 @@ pub type Executive = frame_executive::Executive<
 /// __NOTE:__ THE ORDER IS IMPORTANT.
 pub type Migrations = (Unreleased, Permanent);
 
+use frame_support::traits::{OnRuntimeUpgrade, StorageVersion};
+use pallet_dapp_staking::migration::EXPECTED_PALLET_DAPP_STAKING_VERSION;
+
+// Bump dAppStaking pallet version to 9 (move extrinsic release)
+pub struct BumpDappStakingVersion;
+
+impl OnRuntimeUpgrade for BumpDappStakingVersion {
+    fn on_runtime_upgrade() -> Weight {
+        StorageVersion::new(EXPECTED_PALLET_DAPP_STAKING_VERSION)
+            .put::<pallet_dapp_staking::Pallet<Runtime>>();
+        <Runtime as frame_system::pallet::Config>::DbWeight::get().writes(1)
+    }
+}
```

Comment:
IMO, not the way to do it 🙂 

You have used before `V8ToV9`.
That one checks for versions, and only executes them if current version is `v8`.
It also implicitly bumps up version to `v9`.

So all you should need is _raw_ version of `V8ToV9`, without any additional migration logic.